specs/phase0/p2p-interface.md

```diff
@@ -823,6 +870,14 @@ In Phase 0, peers for attestation subnets will be found using the `attnets` entr
 
 Although this method will be sufficient for early phases of Eth2, we aim to use the more appropriate discv5 topics for this and other similar tasks in the future. ENRs should ultimately not be used for this purpose. They are best suited to store identity, location, and capability information, rather than more volatile advertisements.
 
+### How should fork version be used in practice?
+
+Fork versions are to be manually updated (likely via incrementing or using the less collision-prone git spec hash) at each hard fork. This is to provide native domain separation for signatures as well as to aid in usefulness for identitying peers (via ENRs) and versioning network protocols (e.g. using fork version to naturally version gossipsub topics).
```

Comment:
```suggestion
Fork versions are to be manually updated (likely via incrementing or hash of the spec version) at each hard fork. This is to provide native domain separation for signatures as well as to aid in usefulness for identitying peers (via ENRs) and versioning network protocols (e.g. using fork version to naturally version gossipsub topics).
```
Using the spec or git is limiting, and I wouldn't say it is less collision prone, as not all forks may have the same kind of spec. I think it is better to just give a bare "hash spec version" example.

Reply:
With the genesis domain separation, I'm actually now just considering going back to "increment" with a suggestion to flip a high order bit in the event of a contentious fork.